src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/IncentivePaymentOrchestrator.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+
+namespace SFA.DAS.EmployerIncentives.Functions.PaymentsProcess
+{
+    public static class IncentivePaymentOrchestrator
+    {
+        [FunctionName("IncentivePaymentOrchestrator")]
+        public static async Task RunOrchestrator([OrchestrationTrigger] IDurableOrchestrationContext context, ILogger log)
+        {
+            var collectionPeriod = context.GetInput<string>();
+
+            log.LogInformation($"Incentive Payment process started for collection period {collectionPeriod}", new { collectionPeriod });
```

Comment:
If you delegate the Orchestrator call to a service then you can add the logging as a decorator including logging on error keeping the concerns of logging separate from the functionality.

Reply:
I'm not sure how practical delegating the orchestration would be as you'd need to pass the DurableOrchestrationContext around...I'll stick something in the dev channel.

Reply:
Ok